Classes/Public/Manager/Objects/OCTCall.h

```diff
@@ -11,6 +11,12 @@
 #import "OCTChat.h"
 #import "OCTFriend.h"
 
+typedef NS_ENUM(NSUInteger, OCTCallStatus) {
+    OCTCallStatusPaused,
+    OCTCallStatusActive,
+    OCTCallStatusInActive
```

Comment:
`InActive` is read as "in something". Maybe it would be better to change it to `Inactive`?

Also, you can move it on top and set to 0 explicitly. This way it will be sort of default value.
```
    OCTCallStatusInactive = 0,
    OCTCallStatusPaused,
    OCTCallStatusActive,
```